mp/src/materialsystem/stdshaders/pbr_ps30.fxc

```diff
@@ -251,41 +277,39 @@ float4 main( PS_INPUT i ) : COLOR
 
 	float3 directLighting = 0.0;
 
-	for (uint n = 0; n < NUM_LIGHTS && n < 0; ++n)
+	for (uint n = 0; n < NUM_LIGHTS; ++n) // now lighting works on one model but not on another??
```

Comment:
Is this due to the flashlight thing?

Reply:
nope, it's the same with or without the flashlight on, I just can't get direct lighting to work consistently
perhaps I can fix that in the next update
at least it's better than it was :/